iso15118/shared/security.py

```diff
@@ -813,22 +833,46 @@ def verify_signature(
     # 2. Step: Checking signature value
     logger.debug("Verifying signature value for SignedInfo element")
     pub_key = load_der_x509_certificate(leaf_cert).public_key()
+
+    # The signature value element corresponds to the encryption of the EXI encoded
```

Comment:
That needs to be phrased more clearly:
"The signature value element corresponds to the encryption of the EXI encoded and then hashed signed_info element. 
Signed Info -> EXI encoding -> Hashing -> Encryption with private key => Signature Value
ATTENTION: The hashing and encryption operation is part of the ECDSA (Elliptic Curve Digital Signature Algorithm) operation. That is why we do NOT additionally hash the EXI encoded signed info element."

Reply:
done